src/RealTimeClient.js

```diff
@@ -125,6 +132,43 @@ class RealTimeClient {
           delete this.authenticatedResolve;
         }
         return;
+      case messages.SUBSCRIPTION_START.SUCCESS:
+        {
+          const openRequest = this.openRequests[message.request_id];
+          openRequest.subscriptionStartResolver(message);
+          const handler = openRequest.handler;
+          const subscription = this.subscriptions[messages.subscription_id];
+          if (subscription && subscription.queuedMessages) {
```

Comment:
I might be missing something, but I think you can safely `delete this.openRequests[message.request_id];` here

Reply:
Ah, yes, I should do that.